.travis.yml

```diff
@@ -5,7 +5,10 @@ matrix:
   include:
     - python: "3.6"
     - python: "3.7"
+<<<<<<< HEAD
```

Comment:
This is a text generated by git when solving conflicts. Needs to be removed.

Reply:
Resolve. Please check it again. Thanks.